### DISCUSSION TITLE
Disregard Domains

### DISCUSSION THREAD

**Discussion by KleMic123** [AdguardTeam/AdGuardHome#5003] [Q&A]
Hi,
I'm using philips hue and see incredibly many requests to "diag.meethue.com" in AdGuard.
Is it possible to disregard this domain in AdGuard, because it distorts the statistics?

![Bildschirmfoto 2022-10-05 um 16 10 15](https://user-images.githubusercontent.com/109462287/194081593-662dcf44-4caf-4c16-8f99-8018acb76748.png)

**Comment by emlimap** [comment]
There is an existing feature request for this. See https://github.com/AdguardTeam/AdGuardHome/issues/1717
